heron/metricsmgr/src/thrift/scribe.thrift

```diff
@@ -25,8 +25,8 @@ enum ResultCode
 
 struct LogEntry
 {
-  1:  string category,
-  2:  string message
+  1:  string category;
```

Comment:
1.
https://github.com/facebookarchive/scribe/blob/master/if/scribe.thrift is using "," rather than ";".

2. Good to keep consistent for all: enum ResultCode above is still using ","